### PR TITLE
Show asset decimal

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo-modules/issues/483

### CODE REVIEW

**Comment by chenzhitong** [comment]
- [x] getnep17balance
- [ ] getnep17transfers
- [x] getwalletbalance

File: src/RpcNep17Tracker/Nep17Balance.cs [hunk 1]
```diff
@@ -6,20 +6,21 @@ namespace Neo.Plugins
...
-            writer.WriteVarBytes(Balance.ToByteArray());
+            writer.WriteVarBytes(Balance.Value.ToByteArray());
+            writer.Write(Balance.Decimals);
             writer.Write(LastUpdatedBlock);
         }
 
         void ISerializable.Deserialize(BinaryReader reader)
         {
-            Balance = new BigInteger(reader.ReadVarBytes(512));
+            Balance = new BigDecimal(new BigInteger(reader.ReadVarBytes(512)), reader.ReadByte());
```
Comment (erikzhang) [inline src/RpcNep17Tracker/Nep17Balance.cs:23]: The order of arguments calculation is non-deterministic.

Comment (shargon) [reply src/RpcNep17Tracker/Nep17Balance.cs]: Done

File: src/RpcNep17Tracker/RpcNep17Tracker.cs [hunk 2]
```diff
@@ -192,10 +205,10 @@ void IPersistencePlugin.OnPersist(Block block, DataCache snapshot, IReadOnlyList
                 {
                     if (engine.State.HasFlag(VMState.FAULT)) continue;
                     if (engine.ResultStack.Count <= 0) continue;
-                    nep17BalancePair.Value.Balance = engine.ResultStack.Pop().GetInteger();
+                    nep17BalancePair.Value.Balance = new BigDecimal(engine.ResultStack.Pop().GetInteger(), (byte)engine.ResultStack.Pop().GetInteger());
```
Comment (erikzhang) [inline src/RpcNep17Tracker/RpcNep17Tracker.cs:208]: Same here.

Comment (shargon) [reply src/RpcNep17Tracker/RpcNep17Tracker.cs]: Fixed

**Review by erikzhang — APPROVED** [review]

**Review by chenzhitong — APPROVED** [review]

**Pull request merged**